Changes made

7wlr opened the pull request:
I've changed the distance for the opponent off edge to 9 to prevent the bot from crashing

i've added distance based jumping

and i removed the random strafe because that's to blatant lol

7wlr committed:
added distance based jumping and removed the sudden stop when game is over

7wlr committed:
Update Sumo.kt

Added distance based jumping

7wlr committed:
Update Sumo.kt

removed the crashing

7wlr committed:
Merge remote-tracking branch 'origin/kotlin' into kotlin

# Conflicts:
#	src/main/kotlin/best/spaghetcodes/duckdueller/bot/bots/Sumo.kt

HumanDuck23 commented:
Is this PR complete? The sumo logic doesn't appear to do any strafing anymore now

7wlr committed:
Changed name to Elixir Client removed dodging

horisont1 committed:
Update Sumo.kt

horisont1 committed:
Update Sumo.kt

7wlr commented:
yes it was complete, the strafing was removed because it made the w-taps less accurate and looked really scuffed in general

7wlr committed:
removed dodging entirely from the botbase aswell

horisont1 committed:
Update Sumo.kt

7wlr committed:
Added Boosting

7wlr committed:
Added HUD

7wlr committed:
Made it so you can toggle combo reset's on or off

7wlr committed:
circle method still worked on the bot

7wlr committed:
Added replay clearing to clear yourself from atlas

7wlr committed:
Added FOV changer to make it look more "cinematic"

to turn it off you can just unflip the switch lol

the default settings are set if you just lower your fov to about 70 (default fov) then you should be fine and it looks good imo

ofcourse you can tweak it to your liking :)

7wlr committed:
looked ugly so i removed the chatutil stuff from here

7wlr committed:
added strafing back

7wlr committed:
Create yes

7wlr committed:
added logo for webhook

7wlr committed:
Delete upload directory

7wlr committed:
Custom GUI

Revised Camera
Changed from Duckdueller to WLR
Added lobbymovement to the normal sumo bot

7wlr committed:
Revamped the GUI
Added toggle-able Jumping, Strafing for Sumo

7wlr committed:
Update README.md

7wlr committed:
fix package path

7wlr committed:
Merge remote-tracking branch 'origin/kotlin' into kotlin

7wlr committed:
Spoofed mcmod.info

7wlr committed:
updated README.md

7wlr committed:
added more lobby movement options

7wlr committed:
WLR is now hidden from the handshake to hypixel (aka hidden from the modlist that is sent to hypixel)

7wlr committed:
Gui revamp, Added dynamic breaks, Added hitselecting + hitselecting config

7wlr committed:
Gui revamp, Added dynamic breaks, Added hitselecting + hitselecting config

7wlr committed:
added WPH, CLS, CWS, BWS to the discord webhook

7wlr committed:
Gui revamp

7wlr committed:
Create LICENSE file for WLR

Add WLR License
